docs/it-manual/sre-playbook/email-configuration.md

```diff
@@ -0,0 +1,39 @@
+# Email configuration
```

Comment:
Add this page to https://github.com/civiform/docs/blob/main/docs/SUMMARY.md

Reply:
Done.